staging/src/k8s.io/apiserver/pkg/endpoints/filters/authentication.go

```diff
@@ -58,8 +60,19 @@ var (
 
 	authenticatedAttemptsCounter = metrics.NewCounterVec(
 		&metrics.CounterOpts{
-			Name: "authentication_attempts",
-			Help: "Counter of authenticated attempts.",
+			Name:           "authentication_attempts",
+			Help:           "Counter of authenticated attempts.",
+			StabilityLevel: metrics.ALPHA,
```

Comment:
Are all of the metrics alpha?  There is another one in this file.

Reply:
Yeah, I think so. 
There is no confidence for us to mark a metric with `stable` state when it is born.

By the way, alpha is default stability level for a metric and there is no beta stage(in future maybe )

Reply:
Yes, they all start in alpha.